.semaphore/semaphore.yml

```diff
@@ -162,6 +162,10 @@ promotions:
     pipeline_file: push_images.yml
     auto_promote:
       when: "branch =~ 'master|release-.*'"
+  - name: Release
+    pipeline_file: release.yml
+    auto_promote:
+      when: "tag =~ '^v'"
```

Comment:
Looking at the semaphore docs it looks like this should be `result = 'passed' AND tag =~ '^v1\.'`  according to the example [here](https://docs.semaphoreci.com/using-semaphore/promotions#tagged)

Reply:
right!